typings.d.ts

```diff
@@ -126,6 +126,7 @@ declare namespace alkali {
   }
 
   export class VArray<T = {}> extends Variable<Array<T>> {
+    constructor(value?: Array<T> | Promise<Array<T>> | Variable<Array<T>>)
```

Comment:
I had to add this for some reason, or the T/Array<T> wouldn't be inferred by Variable superclass def.  Not sure this originates from TypeScript or my IDE.